content/beginner/080_scaling/deploy_ca.md

```diff
@@ -104,6 +104,8 @@ aws iam create-policy   \
 Finally, create an IAM role for the cluster-autoscaler Service Account in the kube-system namespace.
 
 ```bash
+export ACCOUNT_ID=$(aws sts get-caller-identity --output text --query Account)
```

Comment:
All these repeatedly steps are doing once at the start of the workshop it is not needed to do it everywhere

Reply:
I'm not sure about that. Some users *(including me)* want to start one "part" of the workshop at a time + other "parts" include the "basic stuff", like this env var. Finally, it doesn't bother those who already have this env var. WDYT?

Reply:
Agree with @allamand we expect echo ACCOUNT_ID, AWS_REGION, AZS, LBC_VERSION, and LBC_CHART_VERSION (Possibly some others) to be define globally to avoid overloading each modules